app/helpers/is-announcement.js

```diff
@@ -2,7 +2,11 @@ import { helper } from '@ember/component/helper';
 import CONSTANTS from 'frontend-kaleidos/config/constants';
 
 function isAnnouncement([type]) {
-  return type.get('uri') === CONSTANTS.AGENDA_ITEM_TYPES.ANNOUNCEMENT;
+  let uri = type;
+  if (typeof type?.get === 'function') {
+    uri = type.get('uri');
+  }
+  return uri === CONSTANTS.AGENDA_ITEM_TYPES.ANNOUNCEMENT;
```

Comment:
Can you use [`get()` from `@ember/object`](https://api.emberjs.com/ember/4.6/functions/@ember%2Fobject/get) instead? That should cover both cases.

Reply:
I think that doesn't work in this situation, right? `mu-search` returns `agendaitem.type: 'URI'`, whereas an Ember Data record has `agendaitem.type.uri: 'URI'`. So for the `mu-search` returned item there's nothing to `get()`.

Reply:
I see, you're right :+1: 
I've added some inline documentation for future me :slightly_smiling_face: